packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.targets

```diff
@@ -155,4 +155,11 @@
   <!-- Import WPF Build logic only when we don't import NETFX's WinFX targets -->
   <Import Project="Microsoft.WinFX.targets" Condition="'$(ImportFrameworkWinFXTargets)' != 'true'"/>
```

Comment:
As you pointed out in the meeting, because the SDK is not referenced explicitly in the project element, the WindowsDesktop SDK is only imported if one or both of UseWPF and UseWindowsForms is true.  Microsoft.WinFX.targets can be scoped to WPF only.

Reply:
Yes, the SDK is imported. This is an existing behaviour. However to be able to build apps a developer must opt-in to Windows Forms and/or WPF.
Also our targets must be conditional to only execute when appropriate.

![image](https://user-images.githubusercontent.com/4403806/126736804-c073d5b2-d90e-4f70-b756-78ab0eb0a044.png)

![image](https://user-images.githubusercontent.com/4403806/126736820-d1bd123c-d3e3-409e-b9ed-85c036b7de57.png)